Source/BugsnagSessionTrackingPayload.m

```diff
@@ -24,33 +26,57 @@ @interface Bugsnag ()
 + (BugsnagClient *)client;
 @end
 
+@interface BugsnagApp ()
++ (BugsnagApp *)appWithDictionary:(NSDictionary *)event config:(BugsnagConfiguration *)config;
+
+- (NSDictionary *)toDict;
+@end
+
+@interface BugsnagSessionTrackingPayload ()
+@property (nonatomic) BugsnagConfiguration *config;
+@end
+
 @implementation BugsnagSessionTrackingPayload
 
-- (instancetype)initWithSessions:(NSArray<BugsnagSession *> *)sessions {
+- (instancetype)initWithSessions:(NSArray<BugsnagSession *> *)sessions config:(BugsnagConfiguration *)config {
     if (self = [super init]) {
         _sessions = sessions;
+        _config = config;
     }
     return self;
 }
 
-
 - (NSMutableDictionary *)toJson {
     NSMutableDictionary *dict = [NSMutableDictionary new];
     NSMutableArray *sessionData = [NSMutableArray new];
-    
+
     for (BugsnagSession *session in self.sessions) {
         [sessionData addObject:[session toJson]];
     }
     BSGDictInsertIfNotNil(dict, sessionData, @"sessions");
     BSGDictSetSafeObject(dict, [Bugsnag client].details, BSGKeyNotifier);
-    
+
+    // app/device data collection currently relies on KSCrash reports,
```

Comment:
nit: we don't need the word "currently".  It does, or it doesn't unless there's future plans or historical info to add.